### ISSUE TITLE
Why is the SDK stuck on 45?

### ISSUE THREAD

**Issue by huawesome** [ChihuahuaChain/chihuahua#84]
I was looking for info on why the chain is still using an old fork of the SDK as seen here: 	

github.com/cosmos/cosmos-sdk => github.com/ChihuahuaChain/cosmos-sdk v0.45.12-chihuahua

I know there was some uncertainty surrounding 46, but I see that Migaloo is launching with it and I am wondering what is preventing chihuahua from using a more recent SDK.

Is it anything more than the burn params/functionality?

**Comment by lydia-pierce** [comment]
The main reason is that we use CosmWasm, which has not released an official version that uses v0.46 of the sdk. Building our own custom fork of CosmWasm that uses v0.46 is a little risky. Notional has released their own custom fork of CosmWasm, but again, it is not an official version, so it is a little risky. And at this point, with CosmWasm releasing a version with v0.47 soon, we may just wait to upgrade to that.

**Comment by huawesome** [comment]
Thanks for the clear response. I was interested to know how you view this and how all the disparate versions fit together.

I see that Juno is using:
```
github.com/CosmWasm/wasmd => github.com/CosmosContracts/wasmd v0.30.0-patch.1
github.com/cosmos/cosmos-sdk v0.45.14
```
which I guess is their own fork of wasmd?

And the recently launched Migaloo chain is running 0.30 w/46, which @faddat helped with is and is indeed running the notional branch of wasmd.
```
github.com/CosmWasm/wasmd => github.com/notional-labs/wasmd v0.30.0-sdk469.4
github.com/cosmos/cosmos-sdk v0.46.11
```

Waiting and doing a clean upgrade when sdk 0.47 is officially compatible with cosmwasm 0.30 sounds like a great plan, and thanks for clarifying!
